### PR TITLE
Add Java code style for IntelliJ 15

### PR DESCRIPTION
**Pull request by garukun**
IntelliJ 15 has a bit of a different XML format than the previous versions. This pull request solely deals with the style updates needed for Java in IntelliJ 15.

### CODE REVIEW

**Comment by danuid** [comment]
Thanks a lot! This also works for on my Idea 14!

File: intellij-15-java-google-style.xml [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+      <emptyLine />
+      <package name="wowza" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="java" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="javax" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="" withSubpackages="true" static="true" />
```
Comment (RawToast) [inline intellij-15-java-google-style.xml:238]: This is a great style, but static imports should be first. I know the current style has them last, but it is incorrect according to the documentation:

See this issue: 
https://github.com/google/styleguide/issues/26

**Comment by rdifalco** [comment]
Can someone merge this?

**Comment by shicks** [comment]
I believe this has been subsumed by #209, which is now merged.

**Pull request closed without merging**